modules/ROOT/pages/index.adoc

```diff
@@ -6,34 +6,23 @@ endif::[]
 
-To alleviate issues arising from differences in these microservices, Anypoint Service Mesh ensures the following application management advantages:
+Your non-MuleSoft microservices might be written using different languages and platforms, or might be deployed in different environments. To alleviate issues arising from differences in these microservices, Anypoint Service Mesh ensures the following application management advantages:
 
 * Provides resilient service-to-service communication using zero-trust policies to handle traffic control, fault tolerance and load balancing 
 * Extends MuleSoft API Management capabilities to any non-Mule service, with policies, security and analytics all applied directly from Anypoint Platform
 * Amplifies reuse of services with discovery of non-Mule services within Exchange, available for use in future projects
 
-
-== Business Solution
-
-Growing business demands for diverse deployment environments introduces a need for rapidly deployable and manageable microservices architecture. In a microservice architecture, you build smaller, targeted services instead of large monolithic applications. These services are faster to build, easier to maintain, and help foster agility. 
-
-In a common enterprise setup, numerous teams are engaged in building services that connect and expose functionalities across all sorts of systems: mainframes, SaaS apps, IoT devices, and so on. The result is a siloed environment, with teams managing these applications in different ways. 
-
-With Anypoint Service Mesh, you can connect your non-Mule applications, data, and devices through APIs in an application network. You can manage, govern, and secure your application network using Anypoint Platform.
-
-When your microservices can exist outside of the MuleSoft ecosystem, built using different languages and deployed to different environments. Anypoint Service Mesh enables you to extend your application network to any type of microservice. 
-
-Most importantly, Anypoint Service Mesh can perform these tasks across all services, independent of the technology upon which it is built. You can discover, manage and secure any type of microservice from a central control plane in the Anypoint Platform.
-
 == Anypoint Service Mesh Architecture
 
-A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job. This allows for providing security to your services. The manner in which these sidecar proxies interact is governed by an overall mixer, where a user can apply different policies and security.
+A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
```

Comment:
```suggestion
A _service mesh_ is an independent architecture layer within a Kubernetes cluster. In a service mesh architecture, services communicate with one another via a sidecar proxy rather than directly.  
```